### ISSUE TITLE
Separate form name and label fields in gpap file

### ISSUE THREAD

**Issue by AndrewM-** [geopaparazzi/geopaparazzi#589]
The 'text' field in the gpap file currently contains the value used as a map label.  By default this is the form name ('section name' in json form def.).  If a field is specified as isLabel, the form name is replaced with contents of the field.  This makes it harder to import gpap data with sqlite database managers as I have to add a new field and populate it with the form name by parsing the json string.  What should be a really simple import process becomes inelegant.  If form name *and* label fields are present, importing directly from gpap files is very simple.

Use case, I am doing a tree survey and the deliverable is a spreadsheet.  I could import the data into a GIS, then copy and paste the attribute table into a spreadsheet.  Process gets tedious if several gpap tree survey files exist.  It is better to import all the gpap data into a database and process the data there, before exporting to a spreadsheet.

**Comment by moovida** [comment]
Yes, I think this makes sense. I will consider this for the next version, in which I will also insert new fields about the gps information.
